first commit

Barosz30 opened the pull request:
- [DEMO LINK](https://barosz30.github.io/layout_stop-watch/)

- [x] Keyframes implemented using from/to + transform with rotate property
- [x] Stopwatch is centered and has correct arrows size
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

mgrinko committed:
Merge pull request #2840 from tiserett/change-checklist

choeqq commented on src/styles/blocks/_stopwatch.scss:
```diff
@@ -0,0 +1,57 @@
+  }
+}
+
+@keyframes circlingaround {
```
could we please do a `camelCase` here? I think then it would be more readable
```suggestion
@keyframes circlingAround {
```

Barosz30 replied:
Ok, done ;)

choeqq reviewed (changes requested):
Looks great, just a little suggestion ⬇️

choeqq reviewed (approved):
LGTM ✅

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #1 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
